dubbo-samples-api/src/main/java/com/alibaba/dubbo/samples/consumer/Application.java

```diff
@@ -18,18 +18,24 @@
 package com.alibaba.dubbo.samples.consumer;
 
 import com.alibaba.dubbo.config.ApplicationConfig;
+import com.alibaba.dubbo.config.ProtocolConfig;
 import com.alibaba.dubbo.config.ReferenceConfig;
 import com.alibaba.dubbo.config.RegistryConfig;
+import com.alibaba.dubbo.rpc.RpcContext;
 import com.alibaba.dubbo.samples.api.GreetingsService;
 
 public class Application {
     public static void main(String[] args) {
         ReferenceConfig<GreetingsService> reference = new ReferenceConfig<>();
         reference.setApplication(new ApplicationConfig("first-dubbo-consumer"));
-        reference.setRegistry(new RegistryConfig("multicast://224.5.6.7:1234"));
+        //reference.setRegistry(new RegistryConfig("multicast://224.5.6.7:1234"));
+        reference.setRegistry(new RegistryConfig("zookeeper://localhost:2181"));
```

Comment:
I want to know where the zk server is. For this sample, users must can start it easily.